scripts/publish/publish.ps1

```diff
@@ -135,6 +135,17 @@ function UploadInstallers($msiFile)
     return 0
 }
 
+function UploadVersionBadge($badgeFile)
+{
+    $fileName = [System.IO.Path]::GetFileName($badgeFile)
+    
+    Write-Host "Uploading the version badge to Latest"
+    UploadFile "dev/Binaries/Latest/$filename" $badgeFile
+    
+    Write-Host "Uploading the version badge to $env:DOTNET_CLI_VERSION"
+    UploadFile "dev/Binaries/$env:DOTNET_CLI_VERSION/$filename" $badgeFile
```

Comment:
Should be simple just prefix the badge file name with `$OSNAME`.  

https://github.com/dotnet/cli/blob/master/scripts/common/_rid.sh#L11